gn3/computations/biweight.R

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
**issue: suggestion:** Could you put this scripts in scripts/biweight.R. Then later, we could package it inside GUIX and call it from there.